CMakeLists.txt

```
@@ -780,6 +780,30 @@ add_custom_target(iree-run-module-test-deps
     "Building IREE run module test targets"
 )
 
+# Convenience target for running IREE tests.
+add_custom_target(iree-run-tests
+  COMMENT
+    "Run IREE unit tests"
+  WORKING_DIRECTORY
+    "${CMAKE_CURRENT_BINARY_DIR}"
+  USES_TERMINAL
+  COMMAND
+    "${CMAKE_COMMAND}" -E echo
+    "The 'iree-run-tests' target is a helper for running ctest. For advanced"
+    "options, build dependencies and invoke ctest independently as in:"
+  COMMAND
+    "${CMAKE_COMMAND}" -E echo
+    "  \\(cd ${CMAKE_CURRENT_BINARY_DIR} \\&\\& cmake --build . --target iree-test-deps \\&\\& ctest --output-on-failure\\)"
```

Recommend the options that specify the build directory with a flag instead of changing into it

```suggestion
    "  \\(cmake --build ${CMAKE_CURRENT_BINARY_DIR} --target iree-test-deps \\&\\& ctest --test-dir ${CMAKE_CURRENT_BINARY_DIR} --output-on-failure\\)"
```

---

I'm going to keep it as is since it produces shorter/more hackable copy-paste stuff. I had it your way and it is worse on the eyes and keyboard.